### PR TITLE
DEVOPS-7343 Add automated build keys

### PR DESCRIPTION
**Pull request by fredsmith**

### CODE REVIEW

**Comment by dsjen** [comment]
@fredsmith — thanks for doing this! This looks to be failing because we require our commits to follow the same semantic versioning as in Paragon, https://github.com/edx/paragon#semantic-release.

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+  - secure: KA9Hgej0FmwlF7gbEmwUBLTaQi/PJa9RfDl9/cYAAhU/2ogQr6E9ZnDW0siWyl/CHauvw38bfx17+Ztf2+5ABDC8V9Id1stvKb+k4prM+s/z2MgzbfJMSgOU3oRja+pKv1FzQSkj+8zGfWuwTs8DSz4jxdW3/im0dd3RtCQBP+ztVn2pauzbYsY7+PjTMKYl7Ow0398id6L0+GTS+jJ+elF5OEP226n0SJJjqUmF2niLzNHNtUmnhk/Rkb0mjwHlzyUggsTlNvKWnwcF/fcEmHprUQ7ygjiRiY2nvlP5UM+kFuVtvYY9Zjmual7MDAW5mdguTw3MpbaY35oqnRYOSis1DhikskhvYogFHxR/0I0XJrnKvx6RJJuf0ME2RLeSQ3u8Oe0hjvCdjFdZTqiv8ma5XKnann/D+a2t3g8qkoFCr0nm4twZtLVECp8AdCioogBwoCPKmVjT4fNtkAOqoSfbCrUV9OfS0oOn+WTU+sb1oF/j7kOvhg/eR8ASXuoim7C9pKF7IuSS4BupkzExmUJi1zr9Ehi8UNkJt1859hkpQQL7BPnRvffXkD406s80DaKpzal5bQmX0ALiTf2ozFzrJnOxR/lF7oRYBlT1OuDQ66hG5j+jSTTYTVnHrzrBjMY09tlC546MA9VMrZEnp0tqqY8GpgOoaoKvWjdfUlc=
```
Comment (efischer19) [inline .travis.yml]: Am I correct in assuming these values are `NPM_TOKEN` and `GH_TOKEN`?

I've deleted the set-on-travis-web-ui values, so we'll see how this works once it merges.

Comment (fredsmith) [reply .travis.yml]: yep

**Comment by efischer19** [comment]
@edx/educator-dahlia  FYI - I'm disabling the "require this branch to be up to date" protection on master.

screen shot 2018-05-02 at 12 00 20 pm
screen shot 2018-05-02 at 12 00 57 pm

It seems pointlessly overzealous, since [github prevents merging PRs that have conflicts anyways](https://help.github.com/articles/about-merge-conflicts/)

**Pull request merged**